WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -343,6 +343,14 @@ extension StorePickerViewController {
 
             dismiss(animated: true) {
                 AppDelegate.shared.authenticatorWasDismissed()
+
+                // grab the tab bar
+                guard let tabBar = AppDelegate.shared.tabBarController else {
+                    return
+                }
+
+                // select My Store
+                tabBar.navigateTo(.myStore, animated: true)
```

Comment:
@mindgraffiti just a quick comment — you can replace everything you added here with `MainTabBarController.switchToMyStoreTab()`